conf/routes

```diff
@@ -17,6 +17,7 @@ GET     /api2/atom/:id                  controllers.Api2.getAtom(id)
 PUT     /api2/atom/:id/asset            controllers.Api2.addAsset(id)
 
 GET     /api/youtube/categories         controllers.YoutubeCategory.list()
+PUT     /api/youtube/videos/:id/update  controllers.YoutubeVideos.update(id)
```

Comment:
It is confusing to use the `atomId` in this case as this endpoint is doing operations on youtube videos. But as you said, changing it to the youtube video id would mean updating any video.